.github/workflows/test_ipex.yml

```diff
@@ -18,8 +18,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        torch-version: ["2.2.0", "2.3.*", "2.4.*"]
-        transformers-version: ["4.39.0", "4.44.*"]
```

Comment:
what's the minimum supported version with the new modeling ?

Reply:
To avoid too many tests and extra efforts for previous issues, we only keep the latest released version torch and the last 1 released version torch.

Reply:
and for transformers it's one version (4.45) ?

Reply:
As, transformers models refactoring so fast, we only keep 1 version for now, it should be 4.46 now, please merge #1024 . Thanks

Reply:
merged, let's pin versions in setup that reflect this limited support